csharp/device/Microsoft.Azure.Devices.Client/Common/SHA.cs

```diff
@@ -0,0 +1,211 @@
+// 
+//  Code by Elze Kool
```

Comment:
As far as know there is no license in the code file.
That class has been around since 2009 as far as know. It has been used in countless NETMF projects since.

Will you reach him officially? Or should I do it?